app/models/exercise_participation.rb

```diff
@@ -0,0 +1,25 @@
+class ExerciseParticipation
+  include Neo4j::ActiveRel
```

Comment:
Neo4j has two model types: `Neo4j::ActiveNode` and `Neo4j::ActiveRel`

`Node`s represent objects, and `Rel`s represent the relationships between objects. (In ActiveRecord this would be a many-to-many join table.)